src/transaction/combine.ts

```diff
@@ -1,42 +1,63 @@
 import * as _ from 'lodash'
 import binary from 'ripple-binary-codec'
-import * as utils from './utils'
 import BigNumber from 'bignumber.js'
+import {ValidationError} from '../common/errors'
 import {decodeAccountID} from 'ripple-address-codec'
 import {validate} from '../common'
 import {computeBinaryTransactionHash} from '../common/hashes'
+import {JsonObject} from 'ripple-binary-codec/dist/types/serialized-type'
+
+/**
+ * The transactions should all be equal EXCEPT for the 'Signers' field. 
+ */
+ function validateAllSignedTransactionsAreEqual(transactions: Array<JsonObject>) {
+  const exampleTransaction = JSON.stringify({...transactions[0], Signers: null})
+  if (transactions.slice(1).some(tx => JSON.stringify({...tx, Signers: null}) !== exampleTransaction)) {
+    throw new ValidationError('txJSON is not the same for all signedTransactions')
+  }
+}
 
 function addressToBigNumber(address) {
   const hex = Buffer.from(decodeAccountID(address)).toString('hex')
   return new BigNumber(hex, 16)
 }
 
+/**
+ * According to the documentation on multi-signatures:
+ * If presented in binary form, the Signers array must be sorted based on 
+ * the numeric value of the signer addresses, with the lowest value first. 
+ * (If submitted as JSON, the submit_multisigned method handles this automatically.)
+ */
 function compareSigners(a, b) {
   return addressToBigNumber(a.Signer.Account).comparedTo(
     addressToBigNumber(b.Signer.Account)
   )
 }
 
+function getATransactionWithAllSigners(transactions: Array<JsonObject>): JsonObject {
+  // Signers must be sorted - see compareSigners for more details
+  const sortedSigners = _.flatMap(transactions, tx => tx.Signers)
+    .filter(signer => signer)
+    .sort(compareSigners)
```

Comment:
check whether you can do `map` followed by `flat` if there's no polyfill for `flatMap`. otherwise, this is a good reason to use lodash or something similar

Reply:
"flat" is also not appearing in the polyfill unfortunately, so I think lodash is probably the best bet

Reply:
Checked out the compiler errors caused by the flatMap configuration change, but it seemed like it would take me a long time to figure out the root cause and debug it all. Didn't seem worth it for removing one instance of lodash in a file which already is using lodash elsewhere. (If you want to see the config change, you can see it in the commit c7f2d28)

Reply:
I mean I'm team lodash lol so I'm ok with this 😹